feature/gribi/encap_frr/README.md

```diff
@@ -275,11 +276,18 @@ NH#202 -> {
   network_instance: "TE_VRF_111"
 }
 
+NHG#200 (Default VRF) {
+  {NH#2000, DEFAULT VRF, weight:1}
+}
+NH#2000 -> {
+    decapsulate_header: OPENCONFIGAFTTYPESDECAPSULATIONHEADERTYPE_IPV4
```

Comment:
We shouldn't need to decap here right?

Reply:
sorry, pasted the wrong changes. fixed now.